### PR TITLE
Lazy loads comments js and css

### PR DESCRIPTION
**Pull request by GlynnPhillips**
Prevents users who never scroll down to the comments from downloading
the extra js and css.

For now just putting this behind a flag so Todd can easily access it to work on the design

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -27,12 +27,16 @@ run:
...
+	nbt build --main-js comments.js --main-sass comments.scss --skip-about --skip-haikro --skip-hash --dev
 	nbt build --dev
 
 build-production:
+	nbt build --main-js comments.js --main-sass comments.scss --skip-about --skip-haikro --skip-hash
 	nbt build
 
+
 watch:
+	@echo " **************************************************\n WATCH IS PARTIALLY BOKEN\n Within next-article there are multiple bundles\n NBT watch will only rebuild main.js and main.scss\n **************************************************"
```
Comment (adambraimbridge) [inline Makefile]: Yeah, watch is also meant to rebuild when partials are edited, but it doesn't.

File: client/components/comments/skeleton.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+		skeleton.appendChild(skeletonInner);
+		skeletonInner.appendChild(skeletonPod);
+		skeletonInner.appendChild(skeletonEdit);
+		skeletonInner.appendChild(skeletonFollow);
+		skeletonInner.appendChild(skeletonSubmit);
+		commentsEl.parentNode.insertBefore(skeleton, commentsEl.nextSibling);
+	}
+}
+
+module.exports = createSkeleton;
```
Comment (adambraimbridge) [inline client/components/comments/skeleton.js]: I've never used <template> tags, but they _might_ be useful here? https://developer.mozilla.org/en/docs/Web/HTML/Element/template ... maybe not.

Comment (laurieboyes) [inline client/components/comments/skeleton.js]: IE :rage1: 

screen shot 2016-04-18 at 14 10 00

File: Makefile (hunk 1 above)
Comment (ironsidevsquincy) [inline Makefile]: `watch` is just for front-end assets. `nodemon` should restart when a partial is changed, but doesn't. not sure why

out of scope, but switching to `n.makefile` would fix this watch

Comment (GlynnPhillips) [inline Makefile]: This is using `n.makefile`, how could this be changed to fix this @ironsidevsquincy?

I think @i-like-robots had a plan for this potentially being replaced in the long term when articles are picked back up

Comment (ironsidevsquincy) [inline Makefile]: sorry, more specifically, n.makefile's `watch`, which uses webpack's watch directly (which has a far more intelligent watching system)

Comment (i-like-robots) [inline Makefile]: Yep, we need to update article to use Webpack instead of OBT. It's not that hard... if I get 15 mins today I will do it!

**Pull request merged**